.github/ISSUE_TEMPLATE/06-release.md

```diff
@@ -4,118 +4,159 @@ about: This template can be used to track the progress of the SDP
 labels: ['epic']
 assignees: ''
-
 ---
 
 <!--
     DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
     This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
 -->
 
-# Stackable Release YY.M.X
-
 > [!IMPORTANT]
 > Important dates:
+>
 > - ... - Release planning
 > - ... - Lock product versions
 > - ... - Final operator-rs release at CoB
 > - ... - Begin bumping operator-rs crates in each operator
-> - ... - Begin release-day tasks (release branches are cut)
+> - ... - Begin release-branching tasks
 > - ... - Target release date (marketable)
 
-## Release checklists
-
-Replace the items in the task lists below with the applicable Pull Requests / Issues
+> [!TIP]
+> Replace the items in the task lists below with the applicable Pull Requests / Issues.
 
-### General Pre-Requisites (before Feature Freeze)
+## Early Pre-release tasks
 
 > [!TIP]
 > These tasks should be done earlier in the process to lessen the burden at Pre-release time.
 
-```[tasklist]
-### Early Pre-release tasks
-- [ ] [Create a "Release Retro" issue](https://github.com/stackabletech/issues/issues/new?template=08-release-retro.md)
+- [ ] [Create a "Release Retro" issue][epr-1]
 - [ ] Define product versions to include in the next release
-- [ ] [Update and release operator-rs workspace members](https://github.com/stackabletech/operator-rs/issues/new?template=release-workspace-members.md)
-- [ ] [Update Rust toolchain of operators](https://github.com/stackabletech/operator-templating/issues/new?template=pre-release.md)
-- [ ] [Update Rust dependencies of operators](https://github.com/stackabletech/issues/issues/new?template=05-pre-release-operator-rust-deps.md)
-- [ ] [Major or Minor Container Images updates](https://github.com/stackabletech/docker-images/issues/new?template=early-pre-release.md)
-- [ ] [Patch Container Images updates](https://github.com/stackabletech/docker-images/issues/new?template=early-pre-release.md)
-```
+- [ ] [Update and release operator-rs workspace members][epr-2]
+- [ ] [Update Rust toolchain of operators][epr-3]
+- [ ] [Update Rust dependencies of operators][epr-4]
+- [ ] [Major or Minor Container Images updates][epr-5]
+- [ ] [Patch Container Images updates][epr-6]
+- [ ] Run `niv update` and test via `make run-dev` (operator-templating)
+
+[epr-1]: https://github.com/stackabletech/issues/issues/new?template=08-release-retro.md
+[epr-2]: https://github.com/stackabletech/operator-rs/issues/new?template=release-workspace-members.md
+[epr-3]: https://github.com/stackabletech/operator-templating/issues/new?template=pre-release.md
+[epr-4]: https://github.com/stackabletech/issues/issues/new?template=05-pre-release-operator-rust-deps.md
+[epr-5]: https://github.com/stackabletech/docker-images/issues/new?template=early-pre-release.md
+[epr-6]: https://github.com/stackabletech/docker-images/issues/new?template=early-pre-release.md
```

Comment:
note: This is the same link as `epr-5`, so we could use that instead in above list.

question: Do we want that "Major and Minor" and "Patch" container updates use the same issue template?

Reply:
For now I'll remove the redundant link, but maybe we do have separate procedures. We'll see in the next release.

Done in cd470bbd382cf0299ed7895ee5344bc1de07416a